oak_crypto/src/hpke/mod.rs

```diff
@@ -66,6 +66,10 @@ impl KeyPair {
         }
     }
 
+    pub fn get_private_key(&self) -> Vec<u8> {
```

Comment:
This should not be returning a Vec. The PrivateKey implementation will zero out the memory of the key on drop. The Vec won't do it, so it will be left around in the deallocated heap memory. This should either return a PrivateKey, or some wrapper around a [u8,32] that will clear the memory on drop.

Reply:
But we will need to copy it into the proto, so may need to have access to bytes anyway